src/libraries/Common/src/Interop/Unix/System.Native/Interop.UnixFileSystemTypes.cs

```diff
@@ -28,6 +28,7 @@ internal enum UnixFileSystemTypes : long
             befs = 0x42465331,
             bdevfs = 0x62646576,
             bfs = 0x1BADFACE,
+            bpf_fs = 0xCAFE4A11,
```

Comment:
Please also update this block https://github.com/dotnet/runtime/blob/db5cf28e502ffd9485e382340dfabc1c6080306d/src/libraries/Native/Unix/System.Native/pal_io.c#L1429

Reply:
good catch, done